launcher/firstLaunch/firstlaunch_moc.cpp

```diff
@@ -279,7 +293,63 @@ QString FirstLaunchView::getHeroesInstallDir()
 	return QString{};
 }
 
-void FirstLaunchView::copyHeroesData(const QString & path)
+void FirstLaunchView::extractGogData()
+{
+#ifndef VCMI_NO_INNOEXTRACT
+	QString filterExe = tr("GOG executable") + " (*.exe)";
+	QString fileExe = QFileDialog::getOpenFileName(this, tr("Select a GOG installer (exe) file..."), QDir::homePath(), filterExe);
```

Comment:
So this works on iOS? Nice!

Reply:
Yes, works without problem. :)
Only not so nice thing is that the Filter doesn't work on iOS (I think it's not supported by iOS).